composer.json

```diff
@@ -20,7 +20,7 @@
     ],
     "require": {
         "php": "^7.2 || ^8.0",
-        "knplabs/knp-menu": "^3.0",
+        "knplabs/knp-menu": "^3.2",
```

Comment:
We should use `^3.2@dev` instead, otherwise we can't get a php8-compatible version

Reply:
I fix this now, but i have mistake in Scrutinizer and i don't know how i can fix this.

Reply:
Okay, tests works, but I have error with phpcs-run and i don't know how to fix it. Do you know how to fix it?

Reply:
it looks like phpcs still doesn't support php 8

Reply:
Okay, so we need to wait for support or we can merge this PR?

Reply:
There's no hurry